views/components/table-of-contents/macro.njk

```diff
@@ -1,10 +1,9 @@
-{% macro tableOfContentLinks(flexibleContent, prefix = null, anchor = 'segment') %}
-    {% if flexibleContent.length > 1 %}
-        <div class="segment-links">
-            {% if prefix %}<p class="u-margin-bottom-s">{{ prefix }}</p>{% endif %}
+{% macro tableOfContents(items, anchor) %}
+    {% if items.length > 1 %}
+        <div class="table-of-contents">
             <ul>
-                {% for block in flexibleContent %}
-                    {% set titleToUse = block.tocTitle or block.title %}
+                {% for item in items %}
```

Comment:
I think I preferred the original version here because it's clearer that it expects to operate on a block of Flexible Content?

Reply:
ah you are right, think we can switch that back. I did that originally because it didn't _always_ operate on flexible content, but I think it does now!